graph.py

```diff
@@ -0,0 +1,19 @@
+graph = { "a" : ["c"],
+          "b" : ["c", "e"],
+          "c" : ["a", "b", "d", "e"],
+          "d" : ["c"],
+          "e" : ["c", "b"],
+          "f" : []
+        }
+
+
```

Comment:
I copy paste it from somewhere. But I guess it is a good notation to distinguish between graph nodes and their connections.